package.json

```diff
@@ -43,7 +43,7 @@
     "generate": "yarn workspace @sap-cloud-sdk/generator run compile && yarn ts-node scripts/generate-test-services.ts",
     "compile": "yarn lerna run compile",
     "doc": "node scripts/generate-docs.js",
-    "version": "node scripts/after-bump.js && yarn doc && git add .",
+    "version": "node scripts/after-bump.js && yarn generate:test-services && yarn doc && git add .",
```

Comment:
should we run lint fix?

Reply:
I added it to the generate command. Not sure why the generate all was different than the others...?